parachain-tracer/src/tracker_storage.rs

```diff
@@ -71,9 +73,37 @@ impl TrackerStorage {
 			.map(|v| v.into_inner().unwrap())
 	}
 
-	pub async fn candidate(&self, candidate_hash: H256) -> Option<CandidateRecord> {
+	pub async fn candidate(&self, block_hash: H256) -> Option<CandidateRecord> {
 		self.storage
-			.storage_read_prefixed(CollectorPrefixType::Candidate(self.para_id), candidate_hash)
+			.storage_read_prefixed(CollectorPrefixType::Candidate(self.para_id), block_hash)
+			.await
+			.map(|v| v.into_inner().unwrap())
+	}
+
+	pub async fn block_timestamp(&self, block_hash: H256) -> Option<Timestamp> {
```

Comment:
Please also add some documentation if possible. E.g. I would really like to see what is the meaning of `block_hash` here.